extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -4,11 +4,46 @@
 
 class Hooks {
 
+	/**
+	 * Register hooks for the extension
+	 */
 	public static function register() {
 		$hooks = new self();
+		\Hooks::register( 'BeforePageDisplay', [ $hooks, 'onBeforePageDisplay' ] );
+		\Hooks::register( 'PageHeaderPageTypePrepared', [ $hooks, 'onPageHeaderPageTypePrepared' ] );
 		\Hooks::register( 'QueryPageUseResultsBeforeRecache', [ $hooks, 'onQueryPageUseResultsBeforeRecache' ] );
 	}
 
+	/**
+	 * Adds assets for TemplateClassification
+	 *
+	 * @param \OutputPage $out
+	 * @param \Skin $skin
+	 *
+	 * @return true
+	 */
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
+		if ( $skin->getUser()->isLoggedIn() && $out->getTitle()->inNamespace( NS_TEMPLATE ) ) {
+			\Wikia::addAssetsToOutput( 'tempate_classification_js' );
+			\Wikia::addAssetsToOutput( 'tempate_classification_scss' );
+		}
+		return true;
+	}
+
+	/**
+	 * @param PageHeaderController $pageHeaderController
+	 * @param Int $ns
+	 */
+	public function onPageHeaderPageTypePrepared( \PageHeaderController $pageHeaderController, $ns ) {
+		if ( $ns === NS_TEMPLATE ) {
```

Comment:
If you pass a `Title` object as an argument you could use `$title->inNamespace( NS_TEMPLATE )` here. :)